main.py

```diff
@@ -85,18 +94,60 @@ def clear():
             settings["vmware"]["path"] = vmwarepath
     else:
         vmwarepath = Path("vmrun")
+if "virtualbox" in hypervisors:
+    # Get path to VirtualBox
+    if "virtualbox" in settings and settings["virtualbox"].get("path"):
+        # VirtualBox path found in settings.json and it's not blank (NoneType/blank strings == False)
+        virtualboxpath = settings["virtualbox"].get("path")
+    elif Path("virtualboxPath.txt").is_file():

```

Comment:
There's no point in adding legacy handling when VirtualBox support doesn't even exist in the current code